packages/dds/map/src/directory.ts

```diff
@@ -1045,15 +1061,15 @@ class SubDirectory extends TypedEventEmitter<IDirectoryEvents> implements IDirec
     /**
      * {@inheritDoc IDirectory.get}
      */
-    public get<T = any>(key: string): T | undefined {
+    public get<T = unknown>(key: string): T | undefined {
```

Comment:
I kept thinking about this case specifically... broadening _the generic parameter_ feels like maybe it wouldn't be breaking so it could also be applied on the types exposed by the package? Because it feels similar to broadening the type of an input parameter to a function. Although, `T` is in the return type as well, so the edge case where a consumer doesn't specify a generic parameter would indeed change the return type from `any` to `unknown`... cc @CraigMacomber as our resident expert on TS, types, and breaking changes. Is that accurate, Craig? (`SubDirectory` is internal to the package, so it's fine to make the change here).

Reply:
I imagine we must have consumers that are taking advantage of the fact that `get` returns `any` and are just calling properties / methods on it without casting to their concrete types.

I do think this is a change we want to make (especially since it will encourage consumers to take advantage of the generic type parameter). But I think it needs to happen in `next`.